apps/badass/package.json

```diff
@@ -39,7 +39,7 @@
     "@sentry/nextjs": "^7.9.0",
     "@sentry/tracing": "^7.9.0",
     "@skillrecordings/config": "workspace:*",
-    "@skillrecordings/convertkit": "workspace:*",
+    "@skillrecordings/convertkit-react-ui": "workspace:*",
```

Comment:
this is the primary change changing the package name to be more specific and imply usage